app/approuter/page.tsx

```diff
@@ -0,0 +1,107 @@
+import Image from "next/image";
```

Comment:
is it possible to keep `app` directory inside `src`?

trying to find a similar example here: https://github.com/vercel/next.js/tree/canary/examples/blog-starter/src/app

(my knowledge with Next.js App Router is very limited)

Reply:
@fahad19 I moved the app directory to the src directory